deploy/kubernetes/elastic-agent-standalone/elastic-agent-standalone-ksm-statefulset-configmap.yaml

```diff
@@ -0,0 +1,251 @@
+# For more information https://www.elastic.co/guide/en/fleet/current/running-on-kubernetes-standalone.html
```

Comment:
I don't easily get why we need these 2 extra files on the same level with `elastic-agent-kustomize` folder. I think we need to clearly explain in a top level README our deployment strategies and their differences. That would make it easy to understand it both for our users but mostly for our developers.

Reply:
Makes sense Chris. I have added comments and also updated Readme

Let me know if sth more needed